### PR TITLE
C#: Fix capitalization of vector properties X and Y

### PR DESCRIPTION
**Pull request by rm-code**

### CODE REVIEW

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 1]
```diff
@@ -297,8 +297,8 @@ the ``_process`` function (make sure it's not indented under the `else`):
 
         Position += velocity * (float)delta;
         Position = new Vector2(
-            x: Mathf.Clamp(Position.x, 0, ScreenSize.x),
-            y: Mathf.Clamp(Position.y, 0, ScreenSize.y)
+            x: Mathf.Clamp(Position.X, 0, ScreenSize.X),
+            y: Mathf.Clamp(Position.Y, 0, ScreenSize.Y)
```
Comment (akien-mga) [inline getting_started/first_2d_game/03.coding_the_player.rst:301]: What about the two `Vector2` fields, shouldn't they be capitalized here too?

Comment (rm-code) [reply getting_started/first_2d_game/03.coding_the_player.rst:301]: @akien-mga These are "just" variable names `new Vector2(x: ..., y: ...)`. Capitalizing them would actually create an issue:
Bildschirmfoto 2023-03-06 um 11 47 02

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks! And congrats for your first merged Godot contribution :tada:

**Comment by mhilbrunner** [comment]
Pushed to `4.0` and `stable`.

**Pull request merged**